ebu_tt_live/ui/user_input_producer/uip.js

```diff
@@ -112,18 +112,24 @@ $(document).ready(function() {
         var minutes = Math.floor(current_media_clock_time / 60000) % 60;
         var seconds = Math.floor(current_media_clock_time / 1000) % 60;
         var clock_str = "";
-        if (hours < 10)
-        clock_str += "0";
-        clock_str += hours.toString();
-        clock_str += ":";
-        if (minutes < 10)
-        clock_str += "0";
-        clock_str += minutes.toString();
-        clock_str += ":";
-        if (seconds < 10)
-        clock_str += "0";
-        clock_str += seconds.toString();
-        $("#running-clock").html(clock_str);
+
+        if (hours < 10) {
+          clock_str += "0";
+          clock_str += hours.toString();
```

Comment:
Wait, what? No, surely we should only be doing the first line if `hours < 10` since that relates to the zero padding...